Sources/Extensions/Foundation/FileManagerExtensions.swift

```diff
@@ -56,5 +56,21 @@ public extension FileManager {
         return nil
     }
 
+    /// SwifterSwift: Save Encodable object to a given URL.
+    ///
+    /// - Parameters:
+    ///   - object: Object of type T that conforms to Encodable protocol.
+    ///   - url: URL to save object to.
+    ///   - encoder: JSONEncoder.
+    /// - Throws: Throws any errors thrown by JSONEncoder or FileManager.
+    public static func encode<T: Encodable>(_ object: T, to url: URL, using encoder: JSONEncoder=JSONEncoder()) throws {
+        do {
+            let encoder = encoder
+            let data = try encoder.encode(object)
+            FileManager.default.createFile(atPath: url.path, contents: data, attributes: nil)
```

Comment:
When this function is not `static` you can call the local `createFile` function.
```suggestion
            createFile(atPath: url.path, contents: data, attributes: nil)
```